src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayPrice = 40;
+  const littleDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days < 3) {
+    return dayPrice * days;
+  } else if (days >= 3 && days <= 6) {
```

Comment:
If you write `return` inside `if` block, the next `else` is useless.

With `return` structure should look like this:

```
if (condition1) {
  return x;
}

if (condition2) {
  return y
}

return z;
```